includes/definitions/ciscon3550.yaml

```diff
@@ -0,0 +1,26 @@
+os: ciscon3550
```

Comment:
Shouldn't the OS be `exalink-fusion`, I checked and the firmware files bear that name.

Reply:
Both would be technically correct in my opinion as Cisco has purchased Exablaze and rebranded their products under the Cisco Nexus branding (https://www.cisco.com/c/en/us/about/corporate-strategy-office/acquisitions/exablaze.html#~tab-products).

That being said, I'm personally in favor of keeping Cisco in the os name if possible; would cisco-exalink-fusion or similar be acceptable or does that seem too verbose?

Reply:
Renamed the OS as advised; valid point.